src/schema/nmdc.yaml

```diff
@@ -2873,3 +2874,16 @@ slots:
     contributors:
       - ORCID:0009-0001-1555-1601 #Anastasiya Prymolenna
       - ORCID:0000-0002-8683-0050 #Montana Smith
+
+  bulk_elect_conductivity:
+    comments:
+      - Compatible with EMSL metadata
```

Comment:
what does this mean? are you implying that any slot that doesn't have this comment isn't compatible with EMSL metadata? What **is** EMSL metadata anyway?

Reply:
This comment can go away. We don't need to include that. EMSL metadata just means it's in the EMSL metadata template but isn't in NMDC. This is a MONet measurement & is needed to support user facility operations.

Reply:
Removed comment